corehq/apps/registration/utils.py

```diff
@@ -282,3 +286,20 @@ def create_30_day_trial(domain_obj):
     )
     trial_subscription.is_active = True
     trial_subscription.save()
+
+
+def create_free_community_plan(domain_obj):
+    community_plan_version = DefaultProductPlan.get_default_plan_by_domain(
+        domain_obj, edition=SoftwarePlanEdition.COMMUNITY
+    )
+    community_account = BillingAccount.objects.get_or_create(
+        name="Free Community Account for %s" % domain_obj.name,
```

Comment:
You can skip calling ```Subscription.new_domain_subscription``` entirely, since having no active subscription is equivalent to having a community subscription.